FindOnPage.md

```diff
@@ -0,0 +1,664 @@
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    interface ICoreWebView2Environment15
+    {
+        /// Creates a new instance of a CoreWebView2FindConfiguration object.
+        /// This configuration object can be used to define parameters for a find operation.
+        /// Returns the newly created FindConfiguration object.
+        CoreWebView2FindConfiguration CreateFindConfiguration();
+    };
+
+runtimeclass CoreWebView2FindConfiguration : [default]ICoreWebView2FindConfiguration {}
+
+    /// Interface defining the find configuration.
+    /// This interface provides the necessary methods and properties to configure a find operation.
+    [com_interface("staging=ICoreWebView2StagingFindConfiguration")]
+    [ms_owner("core", "maxwellmyers@microsoft.com")]
+    [availability("staging")]
+    interface ICoreWebView2FindConfiguration 
+    {
+        /// Gets or sets the find term used for the find operation. Returns the find term.
+        String FindTerm { get; set; };
+
+        /// Gets or sets the direction of the find operation (forward or backward). Returns the find direction.
+        CoreWebView2FindDirection FindDirection { get; set; };
+
+        /// Determines if the find operation is case sensitive. Returns TRUE if the find is case sensitive, FALSE otherwise.
+        /// The locale used to determine case sensitivity is the document's language specified by the HTML lang attribute, or if unspecified then the WebView2's UI locale.
+        /// typically provides a default handling approach, while the document's language settings (e.g., specified using the HTML lang attribute) can override these defaults to apply locale-specific rules. This dual consideration 
+        /// ensures that text is processed in a manner consistent with user expectations and the linguistic context of the content.
+        Boolean IsCaseSensitive { get; set; };
+
+        /// Similar to case sensitivity, word matching also can vary by locale, which may be influenced by both the browser's UI locale and the document's language settings. The browser's UI locale         
+        /// typically provides a default handling approach, while the document's language settings (e.g., specified using the HTML lang attribute) can override these defaults to apply locale-specific rules. This dual consideration 
+        /// ensures that text is processed in a manner consistent with user expectations and the linguistic context of the content.
+        /// ShouldMatchWord determines if only whole words should be matched during the find operation. Returns TRUE if only whole words should be matched, FALSE otherwise.
+        Boolean ShouldMatchWord { get; set; };
+        
+        /// Gets or sets the state of whether all matches are highlighted. 
+        /// Returns TRUE if all matches are highlighted, FALSE otherwise.
+        /// Note: Changes to this property take effect only when StartFind, FindNext, or FindPrevious is called. 
```

Comment:
This claims that Next()/Previous() calls will pick up changes to this property, but below it says "To change the ongoing find session, StartFindAsync must be called again with a new or modified configuration object.", which indicates changing the property between Next() calls is ignored.  Which is correct?